.ado/templates/publish-build-artifacts-for-nuget.yml

```diff
@@ -14,6 +14,20 @@ steps:
       arguments: '-TargetRoot $(Build.ArtifactStagingDirectory)'
     enabled: false
 
+  - task: PublishBuildArtifacts@1
```

Comment:
@Khalef1 this file seems modified by your original branch.
Can your confirm these changes are intended?